tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -118,6 +118,23 @@ def test_concatenate_list(self):
             == "Milk, Eggs, Flour, Water"
         )
 
+    def test_add(self):
```

Comment:
As above, not sure why this is here

Reply:
This test the first transform ("add") which then passes added numbers to "format_ordinal". It is a separate transform but in schemas we use it in tandem with ordinals transform.